CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
-## To be released
+## v0.4.0 - To be released
```

Comment:
I appreciate the effort of keeping the changelog up to date but I would rather have everything inside "to be released". I'm not certain that I will follow this release steps (especially the rust 2018 edition in a patch version)